packages/components/psammead-media-player/src/index.stories.jsx

```diff
@@ -27,3 +27,12 @@ storiesOf('Components|Media Player', module).add('Audio Skin', () => (
     skin="audio"
   />
 ));
+
+storiesOf('Components|Media Player', module).add('Audio Skin AMP', () => (
+  <AmpMediaPlayer
+    isAmp
```

Comment:
The props for the AmpMediaPlayer are different to this. Also you need to add the `ampDecorator`, see line 13 to 21

Reply:
@pjcherriman Done